tests/test_model.py

```diff
@@ -1226,21 +1229,21 @@ def test_calculate_presentation(self):
 
         # This Work starts out with a single CoverageRecord reflecting the
         # work done to generate its initial OPDS entry.
-        [record] = work.coverage_records
-        eq_(WorkCoverageRecord.GENERATE_OPDS_OPERATION, record.operation)
+        [record1, record2] = work.coverage_records
+        # we don't know what order the records came back in, but one of them is an opds generation record.
```

Comment:
When I don't know what order items come back in, I usually sort them. In this case `operation` would work as a sort key. Something like this:

`[choose_edition, generate_opds] = sort(work.coverage_records, key=lambda x: x.operation)`